apps/aeoracle/src/aeo_query_tx.erl

```diff
@@ -103,11 +103,11 @@ origin(#oracle_query_tx{sender = SenderPubKey}) ->
 -spec check(query_tx(), trees(), height()) -> {ok, trees()} | {error, term()}.
```

Comment:
Can comment at line 102 "%% TODO: Check the query? Check TTL relative to Oracle TTL?" be deleted?

Reply:
Yes, you are right, this was something we skipped for the initial release but I fixed it here!